dspec_tester.d

```diff
@@ -24,21 +24,83 @@ struct Config {
     string dmdBinPath = "dmd";
     bool printLineNumbers; // whether line numbers should be shown on errors
 }
-Config config;
+__gshared Config config;
```

Comment:
Do we need the low-level `__gshared` for a casual script?

Reply:
Yeah, we use `parallel` below, so a TLS variable doesn't work (that was the reason for the failures on DAutoTest)
`shared` isn't fun to work with and here I would end up casting whenever accessing `config` :/

Reply:
Annoying or not, I'd be inclined to argue that using `__gshared` for anything other than binding to a C global is a bug. It's incredibly error-prone to do so, because the compile treats it as thread-local even though it isn't. I know that a lot of folks seem to want to use `__gshared` because they either don't like having to use casts with `shared` or they don't understand how to use it correctly, but IMHO, the fact that `__gshared` gets used as often as it does is a serious problem.